### ISSUE TITLE
Where do I find what to put in the "TINYMCE_DEFAULT_CONFIG" variables?

### ISSUE THREAD

**Issue by ZackPlauche** [jazzband/django-tinymce#313]
In the documentation it states that you can go to the website to set these.

The problem is, it's not clear in both documentations where / what specifically to change. 

Can this please be made obvious in the documentation where to get these commands, or even a list of what we can add there?

**Comment by Natim** [comment]
Yes, in the meantime that's things that you can find on the TINYMCE website.
You've got example of the default config here: https://github.com/jazzband/django-tinymce/blob/master/tinymce/settings.py#L6-L21

**Comment by ZackPlauche** [comment]
Ah yes, that's actually where I got the current config from.

But where on the website do you find these settings exactly?

**Comment by Natim** [comment]
On the plugin webpage in the JS panel: https://apps.tiny.cloud/products/advanced-tables/
![image](https://user-images.githubusercontent.com/229453/95755778-18ca9980-0ca5-11eb-9e87-09383fac39d9.png)

**Comment by Natim** [comment]
Also in the documentation https://www.tiny.cloud/docs/demo/
